batch/itests/pom.xml

```diff
@@ -17,6 +17,21 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 		<module>org.jboss.tools.batch.core.itest</module>
 		<module>org.jboss.tools.batch.ui.itest</module>
 	</modules>
+
+	<!-- JBIDE-10933 add javaee site so that these tests can be run against upstream site, rather than expecting dependencies to have been built & resolved locally -->
```

Comment:
That is not enough though, it also should tell to tycho to ignore what is in local repository, to be sure nothing is picked up that was built locally.